src-docs/src/views/steps/steps_complex.js

```diff
@@ -61,19 +61,19 @@ const steps = [
     title: 'The last step has two options',
     children: (
       <EuiText>
```

Comment:
```suggestion
      <EuiText size="s">
```

That will help with sizing separation from the EuiStep title